src/components/learner-credit-management/tests/BudgetDetailPage.test.jsx

```diff
@@ -790,12 +790,58 @@ describe('<BudgetDetailPage />', () => {
     const checkBox = screen.getByTestId('datatable-select-column-checkbox-cell');
     expect(checkBox).toBeInTheDocument();
     userEvent.click(checkBox);
-    await waitFor(() => {
-      expect(screen.getByText('Remind (1)')).toBeInTheDocument();
+    expect(await screen.findByText('Remind (1)')).toBeInTheDocument();
+    expect(await screen.findByText('Cancel (1)')).toBeInTheDocument();
+  });
+
+  it('only allows cancel and not remind in certain states', async () => {
+    useParams.mockReturnValue({
+      budgetId: mockSubsidyAccessPolicyUUID,
+      activeTabKey: 'activity',
     });
-    await waitFor(() => {
-      expect(screen.getByText('Cancel (1)')).toBeInTheDocument();
+    useOfferRedemptions.mockReturnValue({
+      isLoading: false,
+      offerRedemptions: mockEmptyOfferRedemptions,
+      fetchOfferRedemptions: jest.fn(),
+    });
+    useSubsidyAccessPolicy.mockReturnValue({
+      isInitialLoading: false,
+      data: mockAssignableSubsidyAccessPolicy,
+    });
+    useBudgetDetailActivityOverview.mockReturnValue({
+      isLoading: false,
+      data: {
+        contentAssignments: { count: 2 },
+        spentTransactions: { count: 0 },
+      },
     });
+    useBudgetContentAssignments.mockReturnValue({
+      isLoading: false,
+      contentAssignments: {
+        count: 1,
+        results: [
+          {
+            uuid: 'test-uuid',
+            contentKey: mockCourseKey,
+            contentQuantity: -19900,
+            learnerState: 'notifying',
+            recentAction: { actionType: 'reminded', timestamp: '2023-10-27' },
```

Comment:
nit/clarification: Is it possible for the API to return `learnerState: 'notifying'` while the `recentAction` returns a "reminded" action? I believe the `recentAction` would return "assigned" if the `learnerState` is "notifying".